test/interactor_test.rb

```diff
@@ -412,6 +412,11 @@ def initialize(_user, params)
       result.b.must_equal 2
     end
 
+    it 'returns hash values passed in the payload' do
```

Comment:
What do you think about changing the description to `it "returns a hash value that is symbolized shallowly"`?